universal-application-tool-0.0.1/test/export/CsvExporterTest.java

```diff
@@ -0,0 +1,104 @@
+package export;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import models.Applicant;
+import models.Program;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import services.program.*;
+
+public class CsvExporterTest {
+  private static Program fakeProgramWithCsvExport;
+  private ImmutableList<Applicant> fakeApplicants;
+  private Writer writer;
+  private ByteArrayOutputStream inMemoryBytes;
+
+  @BeforeClass
+  public static void createFakeProgram() {
+    ProgramDefinition definition =
+        ProgramDefinition.builder()
+            .setId(1L)
+            .setName("fake program")
+            .setDescription("fake program description")
+            .addExportDefinition(
+                ExportDefinition.builder()
```

Comment:
suggestion (non-blocking): can you factor out the nested builders? Maybe just the `CsvExportConfig`, or maybe the entire `ExportDefinition`. 

1. it is more readable
2. you can share code with the other test

Reply:
Done.